src/js/hooks/cycleCount/useCountStepTable.jsx

```diff
@@ -75,22 +77,19 @@ const useCountStepTable = ({
       };
     }
 
+    if (fieldName === 'inventoryItem_expirationDate') {
+      return {
+        customDateFormat: DateFormat.DD_MMM_YYYY,
+      };
+    }
+
     return {};
   };
 
   const defaultColumn = {
     cell: ({
       getValue, row: { original, index }, column: { id }, table,
     }) => {
-      const isFieldEditable = !original.id.includes('newRow') && id !== 'quantityCounted';
-      // We shouldn't allow users edit fetched data (only quantity counted is editable)
-      if (isFieldEditable || !isEditable) {
-        return (
-          <TableCell className="static-cell-count-step">
-            {getValue()}
-          </TableCell>
-        );
-      }
```

Comment:
Why is it moved down? The main idea of that was to stop the code execution before reaching value and error states. Is everything working on the previous counting step? Can't you call your code before this part without moving it?